wiki/Tournaments/OMF/WC/en.md

```diff
@@ -0,0 +1,485 @@
+---
+tags:
+  - OMF
+  - OMFWC
+---
+
+# Old Map Fantasy World Cup
+
+![OMF World Cup Banner](img/banner.png)
+
+The **Old Map Fantasy World Cup** (***OMFWC***) is a country-based osu! tournament hosted by Kaeldori.
```

Comment:
This should also say what iteration of OMF this is but I am not familiar enough with this tournament series to add it.

```suggestion
The **Old Map Fantasy World Cup** (***OMFWC***) is a country-based 4v4 double-elimination osu! tournament hosted by ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519).
```

Reply:
Well it's kinda a spin off in the series because normally it's a rank restricted tournament where anyone can signup

Reply:
Hmm, I see. If that is the case this should be ```suggestion
The **Old Map Fantasy World Cup** (***OMFWC***) is a country-based 4v4 double-elimination osu! tournament hosted by ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519). It is the first iteration of the Old Map Fantasy World Cup.
```